### ISSUE TITLE
Microsoft.AspNetCore.Identity.EntityFrameworkCore shunld not reference Microsoft.AspNetCore.Identity

### ISSUE THREAD

**Issue by soargon** [aspnet/Identity#2013]
![1](https://user-images.githubusercontent.com/775878/46988744-a7df5e00-d12c-11e8-9455-2dd6c629f0fb.png)
![2](https://user-images.githubusercontent.com/775878/46988745-a9a92180-d12c-11e8-9a9a-6cafc20327c9.png)

**Comment by ajcvickers** [comment]
@soargon Why?

**Comment by soargon** [comment]
Because the project Microsoft.AspNetCore.Identity.EntityFrameworkCore doesn't depend anything on project Microsoft.AspNetCore.Identity.  or this project maybe rename to Microsoft.Extensions.Identity.EntityFrameworkCore?

**Comment by HaoK** [comment]
Yeah this is probably something that we missed updating when we added Extensions.Identity.Core, it should be able to reference that instead
